ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1/commands.proto

```diff
@@ -34,7 +34,8 @@ message Commands {
   string application_id = 3;
 
   // Uniquely identified the command. This identifier should be unique for each new command within an
-  // application domain. It can be used for matching the requests with their respective completions.
+  // application domain, i.e., the triple (command_id, party, application_id) must be unique.
```

Comment:
I would have arranged the members of the tuple in the order of "decreasing scope", i.e., `(application_id, party, command_id)`. But that's a ,matter of taste...

Reply:
Changed, I didn’t really put any thought into the order so no reason to keep it that way :slightly_smiling_face: